wafer/kv/migrations/0001_initial.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auth', '0006_require_contenttypes_0002'),
```

Comment:
Thanks for pointing out migrations.swappable_dependency - that saved me a lot of time digging through the migration documentation.